modules/uhttp/filters_test.go

```diff
@@ -42,14 +42,58 @@ import (
 	"github.com/uber/jaeger-client-go/config"
 )
 
-func TestFilterChain(t *testing.T) {
+func TestDefaultFiltersWithNopHost(t *testing.T) {
+	// setup
 	host := service.NopHost()
+	stats.SetupHTTPMetrics(host.Metrics())
+
+	t.Run("run parallel", func(t *testing.T) {
```

Comment:
Could you set t.Parallel() in the beginning(line 46)?
And testFilter... functions can be easily combined in a map and iterated over:
```
tests := map[string]func(*testing.T, service.Host){
"testFilterChain": testFilterChain,...}
for _, test := range tests {
    test.Run(t, host)
}
```